pkg/daemon/metrics.go

```diff
@@ -34,12 +34,12 @@ var (
 			Help: "logs failed drain",
 		})
 
-	// MCDPivotErr shows errors encountered during pivot
-	MCDPivotErr = prometheus.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Name: "mcd_pivot_err",
-			Help: "errors encountered during pivot",
-		}, []string{"node", "pivot_target", "err"})
+	// MCDPivotErr flags error encountered during pivot
+	MCDPivotErr = prometheus.NewCounterVec(
```

Comment:
You don't need a CounterVec if you have no labels.

```suggestion
	MCDPivotErr = prometheus.NewCounter(
```